routes/erpRoutes/erpApi.js

```diff
@@ -213,8 +213,8 @@ router.route('/offer/summary').get(catchErrors(offerController.summary));
 // //____________________________________________ API for Upload controller _________________
 
 router.route('/public/upload/:model/:fieldId').post(
-  createPublicUpload,
   uploadMiddleware.single('upload'),
```

Comment:
@Aman-Gupta-404 

1. could you add option to add multiply files 
2. Add option to add allowed file , ex only image

Reply:
Hey @salahlalami, so regarding this

 Can you tell me how do I make the option accessible from the backend, like should the options come from request params or from request body?
Secondly, can you tell me if a user is uploading 5 files (one file is of type image, and others of type PDF), and the user has passed the option to accept/allow only image type files, will the server reject all the other types of files and not upload it?

Reply:
1- option is pre-defin config accessible only from the backend , default is single , typeFile is image by deafult, user can't decide . this is generic upload , we will use in different route

2- same this is should have some default , and other is option config is  accessible only from the backend  , user can't change it 

there is no request params or from request body

Reply:
okay, i'll start working on it